engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -161,11 +163,11 @@ public WorldRendererImpl(Context context, GLBufferPool bufferPool) {
         this.backdropProvider = context.get(BackdropProvider.class);
         this.renderingConfig = context.get(Config.class).getRendering();
         this.shaderManager = context.get(ShaderManager.class);
+        vrProvider = OpenVRProvider.getInstance();
         if (renderingConfig.isVrSupport()) {
-            this.vrProvider = new OpenVRProvider();
             context.put(OpenVRProvider.class, vrProvider);
-            if (this.vrProvider.init()) {
-                playerCamera = new OpenVRStereoCamera(this.vrProvider);
+            if (vrProvider.init()) {
+                playerCamera = new OpenVRStereoCamera(vrProvider);
                 currentRenderingStage = RenderingStage.LEFT_EYE;
             } else {
                 playerCamera = new PerspectiveCamera(renderingConfig.getCameraSettings());
```

Comment:
Let's add a comment here to make more explicit what's happening here, i.e.:
"If the vrProvider fails to initialize we assume it is unsupported or there is a problem with the hardware and we fall back on a standard, mono camera."

Feel free to rephrase/reword as needed.